src/base/bittorrent/session.cpp

```diff
@@ -439,6 +438,12 @@ Session::Session(QObject *parent)
     // Soon to be deprecated there
     // More info: https://github.com/arvidn/libtorrent/issues/2251
     pack.set_bool(libt::settings_pack::use_disk_cache_pool, false);
+    // libtorrent 1.1 enable UPnP & NAT-PMP by default
```

Comment:
Typo: `enables`
PS: I probably won't do the release today, so no hurry to fix this ASAP.

Reply:
fixed.